### PR TITLE
[mobile][android] Tutorial corrections and aligning with the android api

### PR DESCRIPTION
**Pull request by IvanKobzarev**

### CODE REVIEW

File: _mobile/android.md [hunk 1]
```diff
@@ -71,9 +71,9 @@ Further in this doc you can find how to rebuild it only for specific list of and
 
 `org.pytorch:pytorch_android_torchvision` - additional library with utility functions for converting `android.media.Image` and `android.graphics.Bitmap` to tensors.
 
-#### 4. Reading static image from android asset
+#### 4. Reading image from android asset
```
Comment (jlin27) [inline _mobile/android.md]: Capitalization "Reading Image from Android Asset"

File: _mobile/android.md [hunk 2]
```diff
@@ -71,9 +71,9 @@ Further in this doc you can find how to rebuild it only for specific list of and
...
 
-All logic happens in [org.pytorch.helloworld.MainActivity](https://github.com/pytorch/android-demo-app/blob/master/HelloWorldApp/app/src/main/java/org/pytorch/helloworld/MainActivity.java#L31-L69).
+All logic happens in [`org.pytorch.helloworld.MainActivity`](https://github.com/pytorch/android-demo-app/blob/master/HelloWorldApp/app/src/main/java/org/pytorch/helloworld/MainActivity.java#L31-L69).
```
Comment (jlin27) [inline _mobile/android.md]: "All [the] logic happens in ..."

File: _mobile/android.md [hunk 3]
```diff
@@ -71,9 +71,9 @@ Further in this doc you can find how to rebuild it only for specific list of and
...
 As a first step we read `image.jpg` to `android.graphics.Bitmap` using standard android api.
```
Comment (jlin27) [inline _mobile/android.md]: "As a first step we read `image.jpg` to `android.graphics.Bitmap` using [the] standard [A]ndroid [API]."

File: _mobile/android.md [hunk 4]
```diff
@@ -90,13 +90,13 @@ Module module = Module.load(assetFilePath(this, "model.pt"));
 Tensor inputTensor = TensorImageUtils.bitmapToFloat32Tensor(bitmap,
```
Comment (jlin27) [inline _mobile/android.md]: Capitalization " Preparing Input"

File: _mobile/android.md [hunk 5]
```diff
@@ -90,13 +90,13 @@ Module module = Module.load(assetFilePath(this, "model.pt"));
...
     TensorImageUtils.TORCHVISION_NORM_MEAN_RGB, TensorImageUtils.TORCHVISION_NORM_STD_RGB);
 ```
-`org.pytorch.torchvision.TensorImageUtils` is part of 'org.pytorch:pytorch_android_torchvision' library.
+`org.pytorch.torchvision.TensorImageUtils` is part of `org.pytorch:pytorch_android_torchvision` library.
 `TensorImageUtils#bitmapToFloat32Tensor` method creates tensor in [torch vision format](https://pytorch.org/docs/stable/torchvision/models.html) using `android.graphics.Bitmap` as a source.
```
Comment (jlin27) [inline _mobile/android.md]: "[The] `TensorImageUtils#bitmapToFloat32Tensor` method creates tensor[s] in [the] [torchvision] format ..."

File: _mobile/android.md [hunk 6]
```diff
@@ -124,13 +124,14 @@ for (int i = 0; i < scores.length; i++) {
 String className = ImageNetClasses.IMAGENET_CLASSES[maxScoreIdx];
 ```
  
-In the following sections you can find detailed explanation of pytorch android api, code walk through for bigger [demo application](https://github.com/pytorch/android-demo-app/tree/master/PyTorchDemoApp), implementation details of api and how to customize and build it from the source. 
+In the following sections you can find detailed explanation of pytorch android api, code walk through for bigger [demo application](https://github.com/pytorch/android-demo-app/tree/master/PyTorchDemoApp),
```
Comment (jlin27) [inline _mobile/android.md]: Line 111: "It's" --> "Its"

Line 127: "...detailed explanation[s] of [PyTorch Android API], code walk-through for [a] bigger demo application..."

File: _mobile/android.md [hunk 7]
```diff
@@ -124,13 +124,14 @@ for (int i = 0; i < scores.length; i++) {
 String className = ImageNetClasses.IMAGENET_CLASSES[maxScoreIdx];
 ```
  
-In the following sections you can find detailed explanation of pytorch android api, code walk through for bigger [demo application](https://github.com/pytorch/android-demo-app/tree/master/PyTorchDemoApp), implementation details of api and how to customize and build it from the source. 
+In the following sections you can find detailed explanation of pytorch android api, code walk through for bigger [demo application](https://github.com/pytorch/android-demo-app/tree/master/PyTorchDemoApp), 
+implementation details of api and how to customize and build it from the source.
```
Comment (jlin27) [inline _mobile/android.md]: "implementation details of [the API], _[remove and]_ how to customize it, and build it from source."

File: _mobile/android.md [hunk 8]
```diff
@@ -124,13 +124,14 @@ for (int i = 0; i < scores.length; i++) {
...
 ```
  
-In the following sections you can find detailed explanation of pytorch android api, code walk through for bigger [demo application](https://github.com/pytorch/android-demo-app/tree/master/PyTorchDemoApp), implementation details of api and how to customize and build it from the source. 
+In the following sections you can find detailed explanation of pytorch android api, code walk through for bigger [demo application](https://github.com/pytorch/android-demo-app/tree/master/PyTorchDemoApp), 
+implementation details of api and how to customize and build it from the source. 
 
 ## Pytorch demo app
 
-Bigger example of application that does image classification from android camera output and text classification you can find in the [same github repo](https://github.com/pytorch/android-demo-app/tree/master/PyTorchDemoApp). 
+A bit bigger example of application, that does image classification from android camera output and text classification, you can find in the [same github repo](https://github.com/pytorch/android-demo-app/tree/master/PyTorchDemoApp).
```
Comment (jlin27) [inline _mobile/android.md]: Suggested Revision: "We have also created another more complex PyTorch Android demo application that does image classification from camera output and text classification in the [same github repo](https://github.com/pytorch/android-demo-app/tree/master/PyTorchDemoApp)."

File: _mobile/android.md [hunk 9]
```diff
@@ -124,13 +124,14 @@ for (int i = 0; i < scores.length; i++) {
...
+In the following sections you can find detailed explanation of pytorch android api, code walk through for bigger [demo application](https://github.com/pytorch/android-demo-app/tree/master/PyTorchDemoApp), 
+implementation details of api and how to customize and build it from the source. 
 
 ## Pytorch demo app
 
-Bigger example of application that does image classification from android camera output and text classification you can find in the [same github repo](https://github.com/pytorch/android-demo-app/tree/master/PyTorchDemoApp). 
+A bit bigger example of application, that does image classification from android camera output and text classification, you can find in the [same github repo](https://github.com/pytorch/android-demo-app/tree/master/PyTorchDemoApp). 
 
-To get device camera output in it uses [android cameraX api](https://developer.android.com/training/camerax
+To get device camera output it uses [android CameraX API](https://developer.android.com/training/camerax
```
Comment (jlin27) [inline _mobile/android.md]: Capitalize "Android"

File: _mobile/android.md [hunk 10]
```diff
@@ -162,7 +163,7 @@ Where `analyzeImage` method processes camera output, `android.media.Image`.
```
Comment (jlin27) [inline _mobile/android.md]: "Where [the] analyzeImage process [the] camera output, android.media.Image. It uses [the] aforementioned..."

File: _mobile/android.md [hunk 11]
```diff
@@ -162,7 +163,7 @@ Where `analyzeImage` method processes camera output, `android.media.Image`.
 
 It uses aforementioned [`TensorImageUtils.imageYUV420CenterCropToFloat32Tensor`](https://github.com/pytorch/pytorch/blob/master/android/pytorch_android_torchvision/src/main/java/org/pytorch/torchvision/TensorImageUtils.java#L90) method to convert `android.media.Image` in `YUV420` format to input tensor.
 
-After getting predicted scores from the model it [finds top K classes](https://github.com/pytorch/android-demo-app/blob/master/PyTorchDemoApp/app/src/main/java/org/pytorch/demo/vision/ImageClassificationActivity.java#L153-L161) with the highest scores and shows on the UI.
+After getting predicted scores from the model it finds top K classes with the highest scores and shows on the UI.
 
 ## Building pytorch android from source
```
Comment (jlin27) [inline _mobile/android.md]: Capitalization "Building PyTorch Android from Source"

File: _mobile/android.md [hunk 12]
```diff
@@ -176,11 +177,14 @@ sh ./scripts/build_pytorch_android.sh
 ```
 
 Its workflow contains several steps:
```
Comment (jlin27) [inline _mobile/android.md]: "Its workflow..." ---> "The workflow..."

File: _mobile/android.md [hunk 13]
```diff
@@ -176,11 +177,14 @@ sh ./scripts/build_pytorch_android.sh
...
-1. Builds libtorch for android for all 4 android abis (armeabi-v7a, arm64-v8a, x86, x86_64)
-2. Creates symbolic links to the results of those builds:
+
+1\. Builds libtorch for android for all 4 android abis (armeabi-v7a, arm64-v8a, x86, x86_64)
```
Comment (jlin27) [inline _mobile/android.md]: Change tenses:
"Builds..." --> "Build"
"Creates..." ---> "Create"
"runs..." ----> "run"

File: _mobile/android.md [hunk 14]
```diff
@@ -244,41 +248,38 @@ org.pytorch.Tensor
 
 If the reader is familiar with pytorch python api, we can think that org.pytorch.Tensor represents torch.tensor, org.pytorch.Module torch.Module<?>, while org.pytorch.IValue represents value of TorchScript variable, supporting all its types. ( https://pytorch.org/docs/stable/jit.html#types )
```
Comment (jlin27) [inline _mobile/android.md]: - Capitalization "pytorch python api" --> "PyTorch Python API"
- Remove the <?> after Module
- Remove spaces before and after parenthesis around the url

Revision: 
"If the reader is familiar with [the PyTorch Python API], we can think [of] org.pytorch.Tensor [representing] torch.tensor, org.pytorch.Module [representing] torch.Module, [and] org.pytorch.IValue [representing the] of TorchScript [the] variable, supporting all its [types](https://pytorch.org/docs/stable/jit.html#types).

File: _mobile/android.md [hunk 15]
```diff
@@ -244,41 +248,38 @@ org.pytorch.Tensor
...
+Content of the Tensor can be provided either as (a) java array  or (b) as `java.nio.DirectByteBuffer` of proper type with native bit order.
 
-In case of (a) proper DirectByteBuffer will be created internally. (b) case has an advantage that user can keep the reference to DirectByteBuffer and change its content in future for the next run, avoiding allocation of DirectByteBuffer for repeated runs.
+In case of (a) proper `DirectByteBuffer` will be created internally. (b) case has an advantage that user can keep the reference to DirectByteBuffer and change its content in future for the next run, avoiding allocation of DirectByteBuffer for repeated runs.
 
-Java’s primitive type byte is signed and java does not have unsigned 8 bit type. For dtype=uint8 api uses byte that will be reinterpretted as uint8 on native side. On java side unsigned value of byte can be read as (byte & 0xFF).
+Java’s primitive type byte is signed and java does not have unsigned 8 bit type. For dtype=uint8 api uses byte that will be reinterpretted as uint8 on native side. On java side unsigned value of byte can be read as `byte & 0xFF`.
 
 #### Tensor content layout
```
Comment (jlin27) [inline _mobile/android.md]: Capitalization "Tensor Content Layout"

File: _mobile/android.md [hunk 16]
```diff
@@ -312,16 +313,15 @@ long[] getDataAsLongArray()
...
-These methods throw IllegalStateException if called for inappropriate dtype.
+These methods throw `IllegalStateException` if called for inappropriate dtype.
 
-### org.pytorch.IValue (IValue)
-[github](https://github.com/pytorch/pytorch/blob/master/android/pytorch_android/src/main/java/org/pytorch/IValue.java)
+### [`org.pytorch.IValue`](https://github.com/pytorch/pytorch/blob/master/android/pytorch_android/src/main/java/org/pytorch/IValue.java)
 
-IValue represents a TorchScript variable that can be one of the supported (by torchscript) types ( https://pytorch.org/docs/stable/jit.html#types ). IValue is a tagged union. For every supported type it has a factory method, method to check the type and a getter method to retrieve a value.
-Getters throw IllegalStateException if called for inappropriate type.
+IValue represents a TorchScript variable that can be one of the supported (by torchscript) types ( [https://pytorch.org/docs/stable/jit.html#types](https://pytorch.org/docs/stable/jit.html#types) ).
```
Comment (jlin27) [inline _mobile/android.md]: hyperlink to the word types
[types](https://pytorch.org/docs/stable/jit.html#types)

File: _mobile/android.md [hunk 17]
```diff
@@ -312,16 +313,15 @@ long[] getDataAsLongArray()
...
-Getters throw IllegalStateException if called for inappropriate type.
+IValue represents a TorchScript variable that can be one of the supported (by torchscript) types ( [https://pytorch.org/docs/stable/jit.html#types](https://pytorch.org/docs/stable/jit.html#types) ). 
+`IValue` is a tagged union. For every supported type it has a factory method, method to check the type and a getter method to retrieve a value.
+Getters throw `IllegalStateException` if called for inappropriate type.
 
-### org.pytorch.Module (Module)
-[github](https://github.com/pytorch/pytorch/blob/master/android/pytorch_android/src/main/java/org/pytorch/Module.java)
+### [`org.pytorch.Module`](https://github.com/pytorch/pytorch/blob/master/android/pytorch_android/src/main/java/org/pytorch/Module.java)
 
 Module is a wrapper of torch.jit.ScriptModule (`torch::jit::script::Module` in pytorch c++ api) which can be constructed with factory method load providing absolute path to the file with serialized TorchScript.
```
Comment (jlin27) [inline _mobile/android.md]: Capitalization "PyTorch C++ API"
"... with [the] factory method"

File: _mobile/android.md [hunk 18]
```diff
@@ -12,7 +12,7 @@ published: true
 
 ## Quick start with a HelloWorld example
```
Comment (jlin27) [inline _mobile/android.md]: Spelling and Capitalization: "Quickstart with a HelloWorld Example"

File: _mobile/android.md [hunk 19]
```diff
@@ -12,7 +12,7 @@ published: true
...
 
-[HelloWorld](https://github.com/pytorch/android-demo-app/tree/master/HelloWorldApp) is a simple image classification application that demonstrates how to use PyTorch android api.
+[HelloWorld](https://github.com/pytorch/android-demo-app/tree/master/HelloWorldApp) is a simple image classification application that demonstrates how to use PyTorch Android API.
 This application runs TorchScript serialized TorchVision pretrained resnet18 model on static image which is packaged inside the app as android asset.
 
 #### 1. Model preparation
```
Comment (jlin27) [inline _mobile/android.md]: Capitalization "Model Preparation"
